### ISSUE TITLE
Forbid equality check with float

### ISSUE THREAD

**Issue by orsinium** [wemake-services/wemake-python-styleguide#1357]
# Rule request

## Thesis

Forbid `==` and `!=` when one side of the expression is a float number. It's ok to handle only some simple cases, something is better than nothing.

```python
# bad
assert a == 1.2
1.2 != a

# good
import math
assert isclose(a, 1.2)
assert not isclose(1.2, a)
```

## Reasoning

It's a hard-to-spot famous bug. 

```
In [1]: .1 + .2 
Out[1]: 0.30000000000000004

In [2]: .1 + .2 == .3
Out[2]: False
```

**Comment by sobolevn** [comment]
Great idea! But, I have opened a similar issue several day ago: #1344
